universal-application-tool-0.0.1/test/support/TestQuestionBank.java

```diff
@@ -57,19 +65,39 @@ public Question applicantAddress() {
     return questionCache.computeIfAbsent(QuestionEnum.APPLICANT_ADDRESS, this::applicantAddress);
   }
 
+  // Checkbox
+  public Question applicantKitchenTools() {
+    return questionCache.computeIfAbsent(
+        QuestionEnum.APPLICANT_KITCHEN_TOOLS, this::applicantKitchenTools);
+  }
+
+  // Dropdown
+  public Question applicantIceCream() {
+    return questionCache.computeIfAbsent(QuestionEnum.APPLICANT_ICE_CREAM, this::applicantIceCream);
+  }
+
   // Name
   public Question applicantName() {
     return questionCache.computeIfAbsent(QuestionEnum.APPLICANT_NAME, this::applicantName);
   }
 
   // Repeated name
   public Question applicantHouseholdMemberName() {
-    // Make sure the next call will have the question ready
```

Comment:
Nit (blocking): I think this is still necessary.

Reply:
Thank you for catching that - didn't mean to delete it! The rebasing and conflicts got real messy. 😆